package.json

```diff
@@ -26,7 +26,7 @@
     "build": "yarn cmd build",
     "watch": "yarn cmd watch",
     "package": "yarn cmd package",
-    "test": "yarn workspace botpress test",
+    "test": "cross-env NATIVE_EXTENSIONS_DIR=build/native-extensions cross-env jest -i --detectOpenHandles -c jest.config.js",
```

Comment:
how is the native extension dir necessary  ? (for sqlite i guess ? )

Reply:
I'm not sure, I just copied the line from the bp package.json

Reply:
The tests run fine locally if I change it to `jest -i --detectOpenHandles -c jest.config.js`, but I'm hesitant to make that change since I don't know if it's fixing anything for someone on a different platform.

Reply:
Let's keep it as is then 👍